### ISSUE TITLE
Add getting started learning flow to website

### ISSUE THREAD

**Issue by lauracowen** [OpenLiberty/openliberty.io#1666]
For new users wanting to know what OL is and how to get started trying it out.

Some ideas we had include:

- Put the Downloads link in the footer so that it can be found (by people as well as google) but is not prominent (as that's not the approach we want to promote).
- Make the Getting Started link in top banner point to a new GS page which guides _new_ users through a brief intro and GS sample, link to GS guide, link to intro topics in the OL docs (eg About Open Liberty). Basically provide a one-page learning path pointing the way through greater levels of detail and learning as required.

@dmuelle @gcharters @NottyCode 
Needs Design work so tagging in @mdenby

**Comment by mdenby** [comment]
I can look at this coming up--just noting that to put pressure on myself :-D

**Comment by steven1046** [comment]
Does the https://openliberty.io/start/ page solve this issue now?

**Comment by lauracowen** [comment]
No. But, if you're needing to work out what work is waiting, you could put this particular issue on ice for now and I'll update it when I get back to it.
